app/views/images/index.html.haml

```diff
@@ -0,0 +1,33 @@
+%div{:id => "addimages"}
+  %a(href='images/new') Add an image
+%br
+- if @images.any?
+  - @images.each do |image|
+    %div{:id => "images"}
+      %h2= image.name
+      = image_tag image.image.url(:thumb)
+      %br
+      = link_to image.name, image_path(image)
+      %br
+      - if current_user == image.user
+        = link_to "Edit image", edit_image_path(image)
+        = link_to "Delete image", image_path(image), method: :delete
+      %br
+      %span.likes_count= image.likes.count
+      = 'like'.pluralize image.likes.count
+      = link_to image_tag('like.png', :width => 30, :height => 30), image_likes_path(image), class: 'likes-link', :id => "likebutton"
+      %br
+      - if image.comments.any?
+        %ul
+          - image.comments.each do |comment|
+            %div{:id => "comments"}
+              %p
+                %strong
+                  = comment.username
+                  %strong :
+                = comment.comments
+      %div{:id => "addcomment"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace